atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -191,4 +203,11 @@ protected void onAcquire(Connection conn) {
             }
         });
     }
+
+    @JsonIgnore
+    @Value.Default
+    @Nullable
+    public String getSqlExceptionOverrideClassname() {
+        return null;
```

Comment:
Can we keep this non-null default of empty to simplify the use sites?

```suggestion
    public String getSqlExceptionOverrideClassname() {
        return "";
```